[SDK-4240] Document examples on customizing push event logic in notification extensions (#6411)
braze-inc/braze-docs: _docs/_developer_guide/platform_integration_guides/swift/push_notifications/push_story.md

class NotificationViewController: BrazePushStory.NotificationViewController {}
```

#### Custom handling push story events

> If you want to custom handle push story notification event logic, inherit `BrazePushStory.NotificationViewController` as above and override the `didReceive` methods as below.

Reviewer: ```suggestion
If you want to implement your own custom logic to handle push story notification events, inherit `BrazePushStory.NotificationViewController` as above and override the `didReceive` methods as below.
```

```swift
import BrazePushStory
import UserNotifications
import UserNotificationsUI

class NotificationViewController: BrazePushStory.NotificationViewController {
  override func didReceive(_ notification: UNNotification) {
    super.didReceive(notification)
    
    // Custom handling logic
  }
  
  override func didReceive(_ response: UNNotificationResponse, completionHandler completion: @escaping (UNNotificationContentExtensionResponseOption) -> Void) {
    super.didReceive(response, completionHandler: completion)
    
    // Custom handling logic
  }
}

> 

Reviewer 2: ```suggestion
```

```

> 

Reviewer 2: ```suggestion
```
